### PR TITLE
[browser] reduce sample re-link

### PR DESCRIPTION
**Pull request by pavelsavara**
to avoid https://github.com/dotnet/runtime/issues/100424

### CODE REVIEW

File: src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/Microsoft.Extensions.Logging.Generators.Roslyn4.0.Tests.csproj [hunk 1]
```diff
@@ -5,7 +5,6 @@
     <DefineConstants>$(DefineConstants);ROSLYN4_0_OR_GREATER</DefineConstants>
     <IsHighAotMemoryUsageTest>true</IsHighAotMemoryUsageTest>
     <EmccLinkOptimizationFlag Condition="'$(ContinuousIntegrationBuild)' == 'true'">-O1</EmccLinkOptimizationFlag>
-    <WasmNativeStrip>false</WasmNativeStrip>
```
Comment (maraf) [inline src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/Microsoft.Extensions.Logging.Generators.Roslyn4.0.Tests.csproj:8]: Setting `EmccLinkOptimizationFlag` will be noop without `WasmNativeStrip` (=> `WasmBuildNative`)

File: src/mono/sample/wasm/browser-bench/Console/Wasm.Console.Bench.Sample.csproj [hunk 2]
```diff
@@ -5,7 +5,6 @@
     <WasmMainJSPath>$(BrowserProjectRoot)\test-main.js</WasmMainJSPath>
     <WasmGenerateRunV8Script>true</WasmGenerateRunV8Script>
     <SuppressTrimAnalysisWarnings>true</SuppressTrimAnalysisWarnings>
-    <WasmNativeStrip>false</WasmNativeStrip>
```
Comment (maraf) [inline src/mono/sample/wasm/browser-bench/Console/Wasm.Console.Bench.Sample.csproj:8]: Will this produce some noise in graphs?

cc @radekdoulik

Comment (pavelsavara) [reply src/mono/sample/wasm/browser-bench/Console/Wasm.Console.Bench.Sample.csproj:8]: I think we are not collecting graphs for V8 console, right ? Anyway, it only could become smaller and faster by this change.

File: src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/Microsoft.Extensions.Logging.Generators.Roslyn4.0.Tests.csproj (hunk 1 above)
Comment (pavelsavara) [reply src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/Microsoft.Extensions.Logging.Generators.Roslyn4.0.Tests.csproj:8]: hehe https://github.com/dotnet/runtime/pull/60701#discussion_r739375291

Comment (pavelsavara) [reply src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/Microsoft.Extensions.Logging.Generators.Roslyn4.0.Tests.csproj:8]: Maybe all this is only needed for AOT  @radekdoulik ?

**Review by maraf — APPROVED** [review]

**Review by radekdoulik — APPROVED** [review]

**Comment by pavelsavara** [comment]
- Known [problem](https://github.com/dotnet/runtime/pull/100408) in `JSImportTest.MissingImport` [Log](https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-100517-merge-23eab0787c63495985/WasmTestOnBrowser-System.Runtime.InteropServices.JavaScript.Tests/1/console.29375e79.log?helixlogtype=result)
- Known [problem](https://github.com/dotnet/runtime/issues/100445) `SignalRClientTests.SignalRPassMessages` [Log](https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-100517-merge-63e2d44e0d0a446591/Workloads-Wasm.Build.Tests.TestAppScenarios.SignalRClientTests/1/console.864e7eff.log?helixlogtype=result)
- Known WASI issue https://github.com/dotnet/runtime/issues/100394

**Pull request merged**